import-service/helpers.js

```diff
@@ -0,0 +1,23 @@
+import { client } from './db';
+
+export const createProducts = async (values) => {
```

Comment:
I guess you can use this function from product-service

Reply:
Function from the product service works with an event but not with values, but yeah, would be good to refactor and reuse it